2020/13xxx/CVE-2020-13671.json

```diff
@@ -1,18 +1,83 @@
 {
-    "data_type": "CVE",
-    "data_format": "MITRE",
-    "data_version": "4.0",
-    "CVE_data_meta": {
-        "ID": "CVE-2020-13671",
-        "ASSIGNER": "cve@mitre.org",
-        "STATE": "RESERVED"
-    },
-    "description": {
-        "description_data": [
-            {
-                "lang": "eng",
-                "value": "** RESERVED ** This candidate has been reserved by an organization or individual that will use it when announcing a new security problem. When the candidate has been publicized, the details for this candidate will be provided."
-            }
-        ]
+  "data_type": "CVE",
+  "data_format": "MITRE",
+  "data_version": "4.0",
+  "CVE_data_meta": {
+    "ID": "CVE-2020-13671",
+    "ASSIGNER": "cve@mitre.org",
+    "STATE": "PUBLIC"
+  },
+  "affects": {
+    "vendor": {
+      "vendor_data": [
+        {
+          "vendor_name": "Drupal ",
+          "product": {
+            "product_data": [
+              {
+                "product_name": "Drupal Core",
+                "version": {
+                  "version_data": [
+                    {
+                      "version_name": "9.0",
+                      "version_affected": "<",
+                      "version_value": "9.0.8",
+                      "platform": ""
+                    },
+                    {
+                      "version_name": "8.9",
+                      "version_affected": "<",
+                      "version_value": "8.9.9",
+                      "platform": ""
+                    },
+                    {
+                      "version_name": "8.8",
+                      "version_affected": "<",
+                      "version_value": "8.8.11",
+                      "platform": ""
+                    },
+                    {
+                      "version_name": "7",
+                      "version_affected": "<",
+                      "version_value": "7.74",
+                      "platform": ""
+                    }
+                  ]
+                }
+              }
+            ]
+          }
+        }
+      ]
     }
-}
\ No newline at end of file
+  },
+  "problemtype": {
+    "problemtype_data": [
+      {
+        "description": [
+          {
+            "lang": "eng",
+            "value": "Remote code execution"
+          }
+        ]
+      }
+    ]
+  },
+  "description": {
+    "description_data": [
+      {
+        "lang": "eng",
+        "value": "This issue affects:\nDrupal  Drupal Core\n9.0 versions prior to 9.0.8;\n8.9 versions prior to 8.9.9;\n8.8 versions prior to 8.8.11;\n7 versions prior to 7.74."
```

Comment:
What is the issue? The description should have at a minimum a vulnerability (i.e., problem type) with sufficient detail as to demonstrate that the vulnerability is unique.